src/main/java/seedu/address/logic/commands/EditTaskCommand.java

```diff
@@ -43,7 +42,7 @@
             + PREFIX_TAG + "Homework";
```

Comment:
Should we change the example to fit better to the profs?😅 Maybe "Do schoolwork CS2103T"?

Reply:
👌